app/Http/Requests/EInvoice/Peppol/AddTaxIdentifierRequest.php

```diff
@@ -18,7 +18,7 @@
 
 class AddTaxIdentifierRequest extends FormRequest
 {
-    public array $vat_regex_patterns = [
+    public static array $vat_regex_patterns = [
```

Comment:
We're using this in https://github.com/invoiceninja/admin-module/pull/28/files#diff-e36c0069110f840c92116836ba1b96313f410c67246def15a92e3ad5f07854f7R33.